final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import re
+from termcolor import colored
+import os
+import colorama
+import logging
+from database_functions import get_news_list_by_date, write_news_to_database
+from parse_rss_functions import get_news_list
+from custom_exceptions import NoInternet, IncorrectURL, IncorrectFilePath, DatabaseConnectionError
+from print_functions import print_news_colorize, print_news_JSON_colorize, print_news, print_news_JSON
```

Comment:
Here and everywhere else:
If you are using all functions inside imported module it is preferable to use just `import`